docs/content/references/components.md

```diff
@@ -28,11 +28,15 @@ Note that for the 5.0.3 release and beyond, the Postgres containers were renamed
 
 | PGO | pgAdmin | pgBackRest | PgBouncer | Postgres | PostGIS |
 |-----|---------|------------|-----------|----------|---------|
+| `5.3.0` | `4.30` | `2.41` | `1.17` | `15,14,13,12,11` | `3.3,3.2,3.1,3.0,2.5,2.4` |
```

Comment:
Either 10 belongs in the Postgres column or it should be removed from `docs/config.toml`, right?
```suggestion
| `5.3.0` | `4.30` | `2.41` | `1.17` | `15,14,13,12,11,10` | `3.3,3.2,3.1,3.0,2.5,2.4` |
```